packages/@react-aria/listbox/src/useOption.ts

```diff
@@ -106,10 +109,10 @@ export function useOption<T>(props: AriaOptionProps, state: ListState<T>, ref: R
 
   let {pressProps} = usePress({...itemProps, isDisabled});
 
-  let {hoverProps} = useHover({
-    isDisabled: isDisabled || !shouldFocusOnHover,
+  let {hoverProps, isHovered} = useHover({
+    isDisabled,
```

Comment:
was this just a mistake? I can't really figure out why it was `isDisabled: isDisabled || !shouldFocusOnHover,`

Reply:
because the hover handler wasn't needed unless `shouldFocusOnHover` is true.

Reply:
sorry, I did this because there was a little problem with ListBox hovering.
Problem resolved by detaching isHovered from here and  adding it to ListBoxOption component itself.

Reply:
it has a problem with ListBox and Picker that uses ListBox component.
If you use useHover in ListBoxOption itself, Picker's focus on hover behavior doesn't work.
Otherwise, ListBox's hover styling doesn't work at all 😭 
Do you have any advice?